CI started failing with error "The following packages have unmet dependencies:"

Any help with this would be appreciated:

Current error output:
```
+ sudo apt-get install --yes --no-install-recommends -V mingw-w64 wine-stable wine32:i386
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libc6:i386 : Depends: libgcc-s1:i386 but it is not going to be installed
 libicu70:i386 : Depends: libgcc-s1:i386 (>= 7) but it is not going to be installed
                 Depends: libstdc++6:i386 (>= 5.2) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Error: Process completed with exit code 100.
```
Can be seen at https://github.com/libexpat/libexpat/actions/runs/5061965535/jobs/9106545994#step:3:665 today but the link will stop working in a few days.

Related: https://github.com/actions/runner-images/issues/4589#issuecomment-1546904883

--- a/.github/workflows/coverage.yml
+++ b/.github/workflows/coverage.yml
@@ -64,9 +64,11 @@ jobs:
             lcov \
             libbsd-dev \
             lzip \
-            moreutils
+            moreutils \
+            ppa-purge
 
         # Install 32bit Wine
+        sudo ppa-purge -y ppa:ubuntu-toolchain-r/test  # to unblock
         sudo apt-get install --yes --no-install-recommends -V \
             mingw-w64 \
             wine-stable \

--- a/.github/workflows/coverage.yml
+++ b/.github/workflows/coverage.yml
@@ -51,7 +51,10 @@ jobs:
         set -x -u
         source /etc/os-release
 
-        sudo apt-get update
+        # Unlock 32bit architecture (for Wine further down)
+        sudo dpkg --add-architecture i386  # for wine32
+        sudo apt-get update  # due to new architecture
+
         sudo apt-get install --yes --no-install-recommends -V \
             cmake \
             docbook2x \
@@ -64,8 +67,6 @@ jobs:
             moreutils
 
         # Install 32bit Wine
-        sudo dpkg --add-architecture i386  # for wine32
-        sudo apt-get update  # due to new architecture
         sudo apt-get install --yes --no-install-recommends -V \
             mingw-w64 \
             wine-stable \

